File: DTC.Domain/Entities/Project.cs
namespace DTC.Domain.Entities
{
    public class Project
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Version { get; set; }
        public DateTime VersionDate { get; set; }
        public string Description { get; set; }
        public Guid StatusId { get; set; }
        public ProjectStatus Status { get; set; }

        public Guid ProjectTypeId { get; set; }
        public ProjectType Type { get; set; }

        public Guid AuthorGroupId { get; set; }
        public AuthorGroup Group { get; set; }
    }
}

File: DTC.AdminTestClient/src/app/core/models/Project.ts
import { Status } from "./Status";

// models/create-project.dto.ts
export interface Project {
  id: number;
  name: string;
  version: string;
  versionDate: string;
  description: string;
  isOpenSource: boolean;
  photoUrl?: string;
  authorGroupId: number;
  status: Status;
  projectTypeId: number;
  beginAge: number;
  endAge: number;
  files: File[];
}
